internal/security/secretstore/vmkencryption_test.go

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package secretstore
+
+import (
+	"errors"
+	"testing"
+
+	. "github.com/edgexfoundry/edgex-go/internal/security/kdf/mocks"
+	. "github.com/edgexfoundry/edgex-go/internal/security/pipedhexreader/mocks"
+	"github.com/edgexfoundry/edgex-go/internal/security/secretstoreclient"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	. "github.com/edgexfoundry/go-mod-secrets/pkg/token/fileioperformer/mocks"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestVMKEncryptionNoIkm tests the no-op path
+func TestVMKEncryptionNoIkm(t *testing.T) {
+	// Arrange
+	fileOpener := &MockFileIoPerformer{}
+	pipedHexReader := &MockPipedHexReader{}
+	lc := &logger.MockLogger{}
+	kdf := &MockKeyDeriver{}
+
+	// Act / Assert
+	vmkEncryption := NewVMKEncryption(fileOpener, pipedHexReader, lc, kdf)
+	encrypting := vmkEncryption.IsEncrypting()
+	assert.False(t, encrypting)
+
+	vmkEncryption.WipeIKM()
+
+	fileOpener.AssertExpectations(t)
+	pipedHexReader.AssertExpectations(t)
+	kdf.AssertExpectations(t)
```

Comment:
Consider using table drive tests to consolidate this duplicated code.

Reply:
For same reason as earlier (each test registers different mocks) these tests do not lend themselves well to table-ization.